### DISCUSSION TITLE
Office 2021安装进度卡住

### DISCUSSION THREAD

**Discussion by zby-02** [YerongAI/Office-Tool#793] [Q&A]
按照教程叙述的代码方式一键安装，进度始终卡在7%，等待超过30分钟也没有进展。网络状态良好，访问其他网页都很快。

**Comment by YerongAI** [comment]
请确保是下载后再部署，**安装文件管理**应该能够显示文件信息，此处可以校验安装文件是否有问题。

![image](https://github.com/YerongAI/Office-Tool/assets/44803668/6931cc04-bf5d-48f0-a787-963a5ec44554)
